internal/controllers/gateway/gateway_controller.go

```diff
@@ -462,10 +462,10 @@ func (r *GatewayReconciler) reconcileUnmanagedGateway(ctx context.Context, log l
 	debug(log, gateway, "gathering the gateway publish service") // this will also be done by the validating webhook, this is a fallback
 	var gatewayServices []*corev1.Service
 	for _, ref := range serviceRefs {
-		r.Log.V(util.DebugLevel).Info("determining service for ref", "ref", ref)
+		r.Log.V(util.DebugLevel).Info("Determining service for ref", "ref", ref)
```

Comment:
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/controllers/gateway/gateway_controller.go#L176
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/controllers/gateway/gateway_controller.go#L196
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/controllers/gateway/gateway_controller.go#L223
These and some others are not changed.